ocaml/xapi/vgpuops.ml

```diff
@@ -108,9 +108,9 @@ let add_pcis_to_vm ~__context host vm pci =
   let value = String.concat "," (List.map Pciops.to_string devs) in
   Db.VM.add_to_other_config ~__context ~self:vm ~key:Xapi_globs.vgpu_pci ~value
 
-let get_free_virtual_function ~__context pf =
+let get_free_virtual_function ~__context vm pf =
```

Comment:
Please can we change the name to "reserve" instead of "get" to reflect the new side-effect? (Same in `Pciops`.)